src/explore-education-statistics-admin/src/services/releaseAncillaryFileService.ts

```diff
@@ -22,19 +24,21 @@ export interface AncillaryFile {
 
 export interface UploadAncillaryFileRequest {
   title: string;
+  summary: string;
   file: File;
 }
 
 export interface AncillaryFileUpdateRequest {
   title: string;
+  summary: string;
 }
 
-function mapFile(file: AncillaryFileInfo): AncillaryFile {
+function mapFile({ name, ...file }: AncillaryFileInfo): AncillaryFile {
   const [size, unit] = file.size.split(' ');
 
   return {
     ...file,
-    title: file.name,
+    title: name,
     filename: file.fileName,
```

Comment:
Ahh the joy of deciding on case for `filename` / `fileName` :)

Reply:
Haha, maybe we'll refactor this all to be consistent one day (but probably never :laughing:)